packages/node_modules/cerebral/src/BaseController.js

```diff
@@ -50,6 +50,8 @@ class BaseController extends FunctionTree {
     })
     this.model = new config.Model(this)
 
+    this.emit('initialized:model')
```

Comment:
@christianalfoni are you ok with new event?

if yes we should also document it in api/controller

Reply:
Jup, discussed this beforehand :)